irohad/ametsuchi/temporary_wsv.hpp

```diff
@@ -51,7 +51,7 @@ namespace iroha {
        */
       virtual bool apply(const model::Transaction &transaction,
                          std::function<bool(const model::Transaction &,
-                                            CommandExecutor &, WsvQuery &)>
+                                            WsvCommand &, WsvQuery &)>
                              function) = 0;
```

Comment:
No, `Context` is basically the transaction, which contains the command. This is required to check the permissions to execute a specific command.